src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -153,30 +145,26 @@ def get_total_count(results):
         id__lte=most_recent_group_first_seen_seven_days_ago.id,
     )
 
+    groups_to_update_count = base_queryset._clone().count()
+
     with sentry_sdk.start_span(description="iterate_chunked_group_ids"):
         for new_group_ids in chunked(
             RangeQuerySetWrapper(
                 base_queryset._clone().values_list("id", flat=True),
                 step=ITERATOR_CHUNK,
                 limit=ITERATOR_CHUNK * 50,
                 result_value_getter=lambda item: item,
-                callbacks=[get_last_id, get_total_count],
+                callbacks=[get_total_count],
             ),
             ITERATOR_CHUNK,
         ):
             run_auto_transition_issues_new_to_ongoing.delay(
                 group_ids=new_group_ids,
             )
 
-    remaining_groups_queryset = base_queryset._clone()
-
     with sentry_sdk.start_span(description="get_remaining_groups") as span:
-        if last_id is not None:
-            span.set_tag("last_id", last_id)
-            span.set_tag("total_count", total_count)
-            remaining_groups_queryset = remaining_groups_queryset.filter(id__gt=last_id)
+        remaining_groups = groups_to_update_count - total_count
```

Comment:
I'm not super familiar with the inner workings of the feature, but from a naive perspective, this seems backwards. Is it correct?

Reply:
Ah, I see. I know that you just merged this, but the next time you're in this code, I think it'd be a public service to either make the names clearer or at least add a comment here explaining the above.